daisy_workflows/build-publish/windows/windows-11-21h2-ent-x64-uefi.publish.json

```diff
@@ -16,18 +16,23 @@
   "PublishProject": "bct-staging-images",
   "ComputeEndpoint": "https://www.googleapis.com/compute/staging_alpha/projects/",
   "DeleteAfter": {{$delete_after}},
-  {{- else -}}
+  {{if eq .environment "internal" -}}
   "WorkProject": {{$work_project}},
   "PublishProject": "google.com:windows-internal",
   "ComputeEndpoint": {{$endpoint}},
   "DeleteAfter": {{$delete_after}},
+  {{- else -}}
+  "WorkProject": {{$work_project}},
+  "PublishProject": "bct-prod-images",
+  "ComputeEndpoint": {{$endpoint}},
+  "DeleteAfter": {{$delete_after}},
   {{- end}}
   {{$guest_features := `["MULTI_IP_SUBNET","UEFI_COMPATIBLE","VIRTIO_SCSI_MULTIQUEUE","GVNIC","WINDOWS"]` -}}
   {{$time := trimPrefix .publish_version "v"}}
   "Images": [
     {
       "Prefix": "windows-11-ent-x64",
-      "Family": "windows-11",
+      "Family": "windows-11-x64",
```

Comment:
windows-11-21h2-x64

I think it would be helpful to include what specific release of Windows 11.

There is no x86 version of windows 11, so I'm not sure x64 is needed, but having it maybe helpful if we have an arm64 in the future.

Reply:
Fixed, agreed on specific release. That's what I was thinking as well- even though there is only a 64 bit version of Win11 naming it this way keeps it consistent with the other versions and prevents us from not having to rename it in the future if we do end up getting an arm version in the future.